apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ThrowableExpectationSamples.kt

```diff
@@ -0,0 +1,63 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ThrowableExpectationSamples {
+
+    @Test
+    fun messageToContain() {
+        expect(RuntimeException("abc")).messageToContain("b")
+
+        fails {
+            expect(RuntimeException("abc")).messageToContain("d")
+        }
+    }
+
+    @Test
+    fun messageFeature() {
+        expect(RuntimeException("abc")).message.toContain("a")
+
+        fails {
+            expect(RuntimeException("abc")).message.toContain("d")
+        }
+    }
+
+    @Test
+    fun message() {
+        expect(RuntimeException("abc")).message {
+            toContain("a")
+        }
+
+        fails {
+            expect(RuntimeException("abc")).message {
+                toContain("d")
+            }
+        }
+    }
+
+    @Test
+    fun causeFeature() {
+        expect(IllegalStateException(IndexOutOfBoundsException())).cause<Throwable>()
```

Comment:
that's not such a good example, cause already narrows the type, no need for the extra `toBeAnInstanceOf`
```suggestion
        expect(IllegalStateException(IndexOutOfBoundsException())).cause<IndexOutOfBoundsException>()
```
I suggest you add a message to the cause exception and check for it